play-json/src/test/scala/com/lightbend/lagom/scaladsl/playjson/PlayJsonSerializerSpec.scala

```diff
@@ -118,6 +118,23 @@ class PlayJsonSerializerSpec extends WordSpec with Matchers {
       }
     }
 
+    "pick up serializers for registry with migration" in withActorSystem(TestRegistry3) { system =>
+
+      val migratedEvent = MigratedEvent(addedField = 2, newName = "some value")
+
+      val serializeExt = SerializationExtension(system)
+      val serializer = serializeExt.findSerializerFor(migratedEvent).asInstanceOf[SerializerWithStringManifest]
+
+      val bytes = serializer.toBinary(migratedEvent)
+      val manifest = serializer.manifest(migratedEvent)
+
+      bytes.isEmpty should be(false)
+
+      val deserialized = serializer.fromBinary(bytes, manifest)
```

Comment:
Would PR  also impact lines 149  and 165 of this tests where manifest was still built manually ?
See `val deserialized = serializer.fromBinary(oldJsonBytes, classOf[MigratedEvent].getName + "#1")` in following tests.